content/15.development_procedure.md

```diff
@@ -66,6 +82,29 @@ The workflow for code review roughly follows this outline:
 One increasing issue with the code review process is ensuring that changes are reviewed with appropriate urgency; larger pull requests tend to languish without review, as the requirements for review necessarily add burden to the maintainers.
 "Bugfix" changes formally require only one reviewer, whereas the `yt` guidelines suggest that larger changes require review from two different team members.
 
+Recently, we have begun utilizing the ["auto-merge" feature](https://github.blog/changelog/2021-02-04-pull-request-auto-merge-is-now-generally-available/) provided by GitHub.
```

Comment:
I think I would reorganise this paragraph to tell the problem first, and then show what solution we adopted. I find it would engage the reader more. Arguably, the problem is also more fundamental than its (current) solution.